valkey.conf

```diff
@@ -386,6 +386,12 @@ databases 16
 # ASCII art logo in startup logs by setting the following option to yes.
 always-show-logo no
 
+# To prevent sensitive user information, such as PII, from being recorded in the 
+# server log file, this feature is enabled by default. If you need to log user 
+# data for debugging or troubleshooting purposes, you can disable this feature
+# by changing the config value to no.
+hide-user-data-from-log yes
```

Comment:
There is no introduction of the feature before "this feature", so it's not clear what it refers to.

```suggestion
# To prevent sensitive user information, such as PII, from being recorded in the
# server log file, user data is hidden from the log by default. If you need to
# log user data for debugging or troubleshooting purposes, you can disable this
# feature by changing the config value to no.
hide-user-data-from-log yes
```